packages/package_info/example/test_driver/package_info_e2e.dart

```diff
@@ -24,6 +24,11 @@ void main() {
       expect(info.buildNumber, '1');
       expect(info.packageName, 'io.flutter.plugins.packageInfoExample');
       expect(info.version, '1.0');
+    } else if (Platform.isMacOS) {
```

Comment:
Given that these are all identical, instead of adding another copy here and below, how about collapsing these to just `if (Platform.isAndroid || Platform.isIOS || Platform.isMacOS)`.

Reply:
Well, it's not really the identical, iOS uses version name "1.0" because this is defined in the `Info.plist` and macOS uses "1.0.0" because it uses the generated `FLUTTER_BUILD_NAME` [as that's the default value](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/ios/xcodeproj.dart#L204) if i understand it correctly, for `Flutter/ephemeral/Flutter-Generated.xcconfig`

also android uses lower case package name `packageinfoexample` vs. ios (and now macos) `packageInfoExample` - I guess it wouldn't be too hard to make it identical though, if that's better?

Reply:
Ah, I missed that. Sounds like this is just something that will need to be revisited if/wher the example version is cleaned up.